can't see freetype2

OS: Gentoo

Issue:
```
$ make
[ 12%] Building C object CMakeFiles/maze4d.dir/glad/src/glad.c.o
[ 25%] Building CXX object CMakeFiles/maze4d.dir/maze4d/Cube.cpp.o
[ 37%] Building CXX object CMakeFiles/maze4d.dir/maze4d/Field.cpp.o
[ 50%] Building CXX object CMakeFiles/maze4d.dir/maze4d/Game.cpp.o
[ 62%] Building CXX object CMakeFiles/maze4d.dir/maze4d/main.cpp.o
/home/eee/tmp/maze4d/maze4d/main.cpp:30:9: warning: #pragma once in main file
   30 | #pragma once
      |         ^~~~
In file included from /home/eee/tmp/maze4d/maze4d/main.cpp:38:
/home/eee/tmp/maze4d/maze4d/UserInterfaceClasses.h:12:10: fatal error: ft2build.h: No such file or directory
   12 | #include <ft2build.h>
      |          ^~~~~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/maze4d.dir/build.make:134: CMakeFiles/maze4d.dir/maze4d/main.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:95: CMakeFiles/maze4d.dir/all] Error 2
make: *** [Makefile:103: all] Error 2
$
```
Solution:
Change in ../CMakeLists.txt line
include_directories(SYSTEM glad/include maze4d)
to
include_directories(SYSTEM glad/include maze4d /usr/include/freetype2)

--- a/.github/workflows/ci-linux.yml
+++ b/.github/workflows/ci-linux.yml
@@ -1,0 +1,20 @@
+name: CI Linux
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+
+    steps:
+    - run: sudo apt install build-essential cmake ninja-build libglfw3-dev libglm-dev libfreetype-dev libfreetype6 libfreetype6-dev
+
+    - uses: actions/checkout@v2
+
+    - run: mkdir builddir
+    - run: cmake ..
+      working-directory: ./builddir
+    - run: make
+      working-directory: ./builddir

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -7,6 +7,7 @@ cmake_policy(SET CMP0072 NEW)
 find_package(OpenGL REQUIRED)
 find_package(glfw3 3.3 REQUIRED)
 find_package(glm REQUIRED)
+find_package(Freetype REQUIRED)
 
 
 add_executable(${PROJECT_NAME}
@@ -17,7 +18,15 @@ add_executable(${PROJECT_NAME}
     maze4d/main.cpp
     maze4d/Maze.cpp
     maze4d/Utils.cpp
+    maze4d/HotkeysMenu.cpp
+    maze4d/MainMenu.cpp
+    maze4d/MainMenuSettings.cpp
+    maze4d/MainUiController.cpp
+    maze4d/Player.cpp
+    maze4d/SettingsMenu.cpp
+    maze4d/SettingsMenuGameStart.cpp
+    maze4d/Texture.cpp
+    maze4d/UserInterfaceClasses.cpp
 )
-include_directories(SYSTEM glad/include maze4d)
-
-target_link_libraries(${PROJECT_NAME} gcc_s c glfw OpenGL::GL ${CMAKE_DL_LIBS})
+include_directories(SYSTEM ${FREETYPE_INCLUDE_DIRS} glad/include maze4d)
+target_link_libraries(${PROJECT_NAME} gcc_s c glfw OpenGL::GL ${CMAKE_DL_LIBS} ${FREETYPE_LIBRARIES})

--- a/README.md
+++ b/README.md
@@ -1,6 +1,8 @@
 # maze4d
 First person four-dimensional maze game, visualized by 3D cross-sections.
 
+[![CI Linux](https://github.com/IronLynx/maze4d/actions/workflows/ci-linux.yml/badge.svg)](https://github.com/IronLynx/maze4d/actions/workflows/ci-linux.yml)
+
 ## Build
 ### Windows
 Just clone the repository, and build `maze4d.sln` in Visual Studio 2015.

--- a/maze4d/UserInterfaceClasses.h
+++ b/maze4d/UserInterfaceClasses.h
@@ -162,7 +162,7 @@ protected:
 	std::string ParamTextForDisplay(float value, CFG_VALUE_TYPE valueType);
 
 	void ReloadParams();
-	void SettingsMenu::SwitchEditMode();
+	void SwitchEditMode();
 	void RecalculateButtonWidth();
 };
 

